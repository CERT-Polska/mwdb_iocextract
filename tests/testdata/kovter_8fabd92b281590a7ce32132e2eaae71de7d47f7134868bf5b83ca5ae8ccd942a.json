{
    "b_dll": false,
    "cptm": "30",
    "debug": false,
    "dl_sl": false,
    "dnet32": "http://download.microsoft.com/download/0/8/c/08c19fa4-4c4f-4ffb-9d6c-150906578c9e/NetFx20SP1_x86.exe",
    "dnet64": "http://download.microsoft.com/download/9/8/6/98610406-c2b7-45a4-bdc3-9db1b1c5f7e2/NetFx20SP1_x64.exe",
    "elg": true,
    "fl_fu": "https://fpdownload.macromedia.com/get/flashplayer/current/licensing/win/install_flash_player_24_active_x.exe",
    "flags": {
        "DD10D": true,
        "DD11D": false,
        "DD12D": true,
        "DD13D": true,
        "DD14D": true,
        "DD15D": true,
        "DD16D": true,
        "DD17D": false,
        "DD1D": true,
        "DD2D": true,
        "DD3D": true,
        "DD4D": true,
        "DD5D": false,
        "DD6D": true,
        "DD7D": true,
        "DD8D": true,
        "DD9D": true,
        "al": "http://185.117.72.90/upload.php"
    },
    "nonul": "http://185.117.72.90/upload2.php",
    "pshell2k3x32": "http://download.microsoft.com/download/1/1/7/117FB25C-BB2D-41E1-B01E-0FEB0BC72C30/WindowsServer2003-KB968930-x86-ENG.exe",
    "pshell2k3x64": "http://download.microsoft.com/download/B/D/9/BD9BB1FF-6609-4B10-9334-6D0C58066AA7/WindowsServer2003-KB968930-x64-ENG.exe",
    "pshellvistax32": "http://download.microsoft.com/download/A/7/5/A75BC017-63CE-47D6-8FA4-AFB5C21BAC54/Windows6.0-KB968930-x86.msu",
    "pshellvistax64": "http://download.microsoft.com/download/3/C/8/3C8CF51E-1D9D-4DAA-AAEA-5C48D1CD055C/Windows6.0-KB968930-x64.msu",
    "pshellxp": "http://download.microsoft.com/download/E/C/E/ECE99583-2003-455D-B681-68DB610B44A4/WindowsXP-KB968930-x86-ENG.exe",
    "public_key": {
        "e": 65537,
        "n": "20717578436666370206990156461786566788132748458910865354994919388630407187082788932551065567891365033974994995141358277530021944793516607142737605543772104350635734672485498640041982499636009940196953103877199811371834197299886690010229547993815721647414299018829914480336700775760032044922438942690008663278856440487164946050309668972730239620373400036156807226902415414689227139343695179004305146177952041410093920067335850237232148134221904306706694425837140102211178161590920721365317540938040383023194954613997204876850415109848188765254167924483000246775174171501733414326729845936854172715365200925796295269097"
    },
    "raw_cfg": {
        "in-blob": "76961b420702604c8ff24be94c5cd14eeaaf40cf829913f8ec9c50ac66ea17ee"
    },
    "rc4key": "a7887cc809cf0d4df17fc5dafd03e4e7",
    "type": "kovter",
    "urls": [
        {
            "cnc": "150.219.156.87",
            "port": 80
        },
        {
            "cnc": "59.34.180.235",
            "port": 38232
        },
        {
            "cnc": "169.136.157.237",
            "port": 80
        },
        {
            "cnc": "62.220.112.204",
            "port": 443
        },
        {
            "cnc": "18.56.29.198",
            "port": 80
        },
        {
            "cnc": "218.216.127.77",
            "port": 80
        },
        {
            "cnc": "21.41.239.107",
            "port": 80
        },
        {
            "cnc": "100.166.63.24",
            "port": 80
        },
        {
            "cnc": "106.192.26.7",
            "port": 80
        },
        {
            "cnc": "195.69.139.52",
            "port": 443
        },
        {
            "cnc": "243.69.73.16",
            "port": 80
        },
        {
            "cnc": "137.234.227.8",
            "port": 80
        },
        {
            "cnc": "55.29.95.39",
            "port": 80
        },
        {
            "cnc": "125.138.46.188",
            "port": 80
        },
        {
            "cnc": "191.38.99.216",
            "port": 443
        },
        {
            "cnc": "17.155.112.156",
            "port": 80
        },
        {
            "cnc": "129.143.21.202",
            "port": 8080
        },
        {
            "cnc": "32.84.137.4",
            "port": 443
        },
        {
            "cnc": "191.59.120.31",
            "port": 80
        },
        {
            "cnc": "255.155.235.46",
            "port": 80
        },
        {
            "cnc": "141.236.125.239",
            "port": 80
        },
        {
            "cnc": "169.1.96.26",
            "port": 443
        },
        {
            "cnc": "48.155.43.68",
            "port": 443
        },
        {
            "cnc": "202.100.184.83",
            "port": 80
        },
        {
            "cnc": "20.19.162.140",
            "port": 80
        },
        {
            "cnc": "3.140.205.238",
            "port": 80
        },
        {
            "cnc": "37.123.165.161",
            "port": 443
        },
        {
            "cnc": "106.74.107.202",
            "port": 80
        },
        {
            "cnc": "8.249.254.51",
            "port": 80
        },
        {
            "cnc": "99.252.161.28",
            "port": 80
        },
        {
            "cnc": "9.48.98.170",
            "port": 80
        },
        {
            "cnc": "147.173.72.96",
            "port": 443
        },
        {
            "cnc": "1.132.22.166",
            "port": 443
        },
        {
            "cnc": "129.16.111.236",
            "port": 80
        },
        {
            "cnc": "210.243.212.209",
            "port": 8080
        },
        {
            "cnc": "18.53.35.179",
            "port": 80
        },
        {
            "cnc": "114.187.128.212",
            "port": 80
        },
        {
            "cnc": "60.103.18.131",
            "port": 80
        },
        {
            "cnc": "172.132.76.194",
            "port": 443
        },
        {
            "cnc": "113.67.58.224",
            "port": 80
        },
        {
            "cnc": "20.179.35.232",
            "port": 80
        },
        {
            "cnc": "73.249.184.108",
            "port": 80
        },
        {
            "cnc": "9.222.103.137",
            "port": 443
        },
        {
            "cnc": "204.197.26.221",
            "port": 443
        },
        {
            "cnc": "224.138.203.45",
            "port": 80
        },
        {
            "cnc": "244.157.143.47",
            "port": 80
        },
        {
            "cnc": "190.67.48.224",
            "port": 80
        },
        {
            "cnc": "180.42.36.109",
            "port": 80
        },
        {
            "cnc": "208.118.116.55",
            "port": 80
        },
        {
            "cnc": "4.195.63.225",
            "port": 25900
        },
        {
            "cnc": "32.107.214.76",
            "port": 80
        },
        {
            "cnc": "203.233.71.250",
            "port": 443
        },
        {
            "cnc": "6.61.150.230",
            "port": 80
        },
        {
            "cnc": "75.16.138.183",
            "port": 80
        },
        {
            "cnc": "90.45.25.145",
            "port": 443
        },
        {
            "cnc": "63.149.238.126",
            "port": 80
        },
        {
            "cnc": "249.158.225.208",
            "port": 80
        },
        {
            "cnc": "156.211.224.150",
            "port": 43912
        },
        {
            "cnc": "229.210.208.203",
            "port": 80
        },
        {
            "cnc": "27.219.195.210",
            "port": 80
        },
        {
            "cnc": "30.255.153.175",
            "port": 80
        },
        {
            "cnc": "216.69.26.86",
            "port": 80
        },
        {
            "cnc": "182.180.65.173",
            "port": 443
        },
        {
            "cnc": "197.45.165.116",
            "port": 443
        },
        {
            "cnc": "79.101.37.210",
            "port": 80
        },
        {
            "cnc": "12.25.99.130",
            "port": 80
        },
        {
            "cnc": "50.56.242.72",
            "port": 8080
        },
        {
            "cnc": "187.108.195.8",
            "port": 8080
        },
        {
            "cnc": "212.219.93.114",
            "port": 443
        },
        {
            "cnc": "138.4.86.20",
            "port": 8080
        },
        {
            "cnc": "132.247.145.147",
            "port": 443
        },
        {
            "cnc": "209.159.149.156",
            "port": 443
        },
        {
            "cnc": "202.191.121.100",
            "port": 443
        },
        {
            "cnc": "20.243.155.227",
            "port": 443
        },
        {
            "cnc": "53.128.177.21",
            "port": 8080
        },
        {
            "cnc": "235.250.233.187",
            "port": 80
        },
        {
            "cnc": "35.214.161.230",
            "port": 443
        },
        {
            "cnc": "34.5.168.186",
            "port": 443
        },
        {
            "cnc": "210.147.248.235",
            "port": 443
        },
        {
            "cnc": "254.220.78.226",
            "port": 47857
        },
        {
            "cnc": "130.99.108.151",
            "port": 443
        },
        {
            "cnc": "87.145.98.19",
            "port": 80
        },
        {
            "cnc": "133.232.247.107",
            "port": 80
        },
        {
            "cnc": "25.111.58.211",
            "port": 80
        },
        {
            "cnc": "13.102.27.247",
            "port": 80
        },
        {
            "cnc": "205.246.43.28",
            "port": 80
        },
        {
            "cnc": "229.157.60.81",
            "port": 8080
        },
        {
            "cnc": "180.168.197.23",
            "port": 80
        },
        {
            "cnc": "29.156.163.20",
            "port": 443
        },
        {
            "cnc": "53.44.118.111",
            "port": 80
        },
        {
            "cnc": "123.100.180.115",
            "port": 43893
        },
        {
            "cnc": "129.105.221.156",
            "port": 443
        },
        {
            "cnc": "194.58.126.20",
            "port": 80
        },
        {
            "cnc": "50.188.52.73",
            "port": 80
        },
        {
            "cnc": "80.228.26.99",
            "port": 80
        },
        {
            "cnc": "143.97.189.141",
            "port": 32240
        },
        {
            "cnc": "241.174.170.164",
            "port": 28721
        },
        {
            "cnc": "20.129.203.86",
            "port": 80
        },
        {
            "cnc": "6.211.88.116",
            "port": 80
        },
        {
            "cnc": "20.168.78.137",
            "port": 80
        },
        {
            "cnc": "163.91.30.241",
            "port": 27879
        },
        {
            "cnc": "174.120.121.230",
            "port": 39788
        },
        {
            "cnc": "39.144.13.86",
            "port": 80
        },
        {
            "cnc": "142.34.249.209",
            "port": 443
        },
        {
            "cnc": "204.42.154.209",
            "port": 80
        },
        {
            "cnc": "66.32.198.58",
            "port": 80
        },
        {
            "cnc": "105.149.112.90",
            "port": 80
        },
        {
            "cnc": "238.9.247.103",
            "port": 80
        },
        {
            "cnc": "141.127.109.227",
            "port": 35000
        },
        {
            "cnc": "250.5.29.204",
            "port": 80
        },
        {
            "cnc": "232.245.197.186",
            "port": 80
        },
        {
            "cnc": "8.218.248.66",
            "port": 80
        },
        {
            "cnc": "97.215.155.187",
            "port": 80
        },
        {
            "cnc": "138.196.78.240",
            "port": 80
        },
        {
            "cnc": "173.126.49.27",
            "port": 443
        },
        {
            "cnc": "84.22.102.112",
            "port": 80
        },
        {
            "cnc": "145.89.215.87",
            "port": 8080
        },
        {
            "cnc": "10.94.237.3",
            "port": 80
        },
        {
            "cnc": "25.100.119.180",
            "port": 443
        },
        {
            "cnc": "206.63.226.28",
            "port": 80
        },
        {
            "cnc": "149.201.173.198",
            "port": 80
        },
        {
            "cnc": "15.26.248.116",
            "port": 8080
        },
        {
            "cnc": "218.5.226.178",
            "port": 80
        },
        {
            "cnc": "245.187.185.226",
            "port": 80
        },
        {
            "cnc": "90.251.34.209",
            "port": 443
        },
        {
            "cnc": "65.159.238.36",
            "port": 443
        },
        {
            "cnc": "30.184.131.202",
            "port": 443
        },
        {
            "cnc": "103.216.152.95",
            "port": 80
        },
        {
            "cnc": "34.58.82.4",
            "port": 80
        },
        {
            "cnc": "249.167.103.219",
            "port": 47074
        },
        {
            "cnc": "192.214.135.145",
            "port": 80
        },
        {
            "cnc": "199.48.116.234",
            "port": 80
        },
        {
            "cnc": "163.109.92.34",
            "port": 42753
        }
    ],
    "version": 24
}