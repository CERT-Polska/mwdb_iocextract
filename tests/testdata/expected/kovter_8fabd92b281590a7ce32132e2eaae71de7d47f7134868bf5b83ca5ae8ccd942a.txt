RsaKey n=20717578436666370206990156461786566788132748458910865354994919388630407187082788932551065567891365033974994995141358277530021944793516607142737605543772104350635734672485498640041982499636009940196953103877199811371834197299886690010229547993815721647414299018829914480336700775760032044922438942690008663278856440487164946050309668972730239620373400036156807226902415414689227139343695179004305146177952041410093920067335850237232148134221904306706694425837140102211178161590920721365317540938040383023194954613997204876850415109848188765254167924483000246775174171501733414326729845936854172715365200925796295269097 e=65537
Key rc4:a7887cc809cf0d4df17fc5dafd03e4e7
NetLoc 150.219.156.87:80
NetLoc 59.34.180.235:38232
NetLoc 169.136.157.237:80
NetLoc 62.220.112.204:443
NetLoc 18.56.29.198:80
NetLoc 218.216.127.77:80
NetLoc 21.41.239.107:80
NetLoc 100.166.63.24:80
NetLoc 106.192.26.7:80
NetLoc 195.69.139.52:443
NetLoc 243.69.73.16:80
NetLoc 137.234.227.8:80
NetLoc 55.29.95.39:80
NetLoc 125.138.46.188:80
NetLoc 191.38.99.216:443
NetLoc 17.155.112.156:80
NetLoc 129.143.21.202:8080
NetLoc 32.84.137.4:443
NetLoc 191.59.120.31:80
NetLoc 255.155.235.46:80
NetLoc 141.236.125.239:80
NetLoc 169.1.96.26:443
NetLoc 48.155.43.68:443
NetLoc 202.100.184.83:80
NetLoc 20.19.162.140:80
NetLoc 3.140.205.238:80
NetLoc 37.123.165.161:443
NetLoc 106.74.107.202:80
NetLoc 8.249.254.51:80
NetLoc 99.252.161.28:80
NetLoc 9.48.98.170:80
NetLoc 147.173.72.96:443
NetLoc 1.132.22.166:443
NetLoc 129.16.111.236:80
NetLoc 210.243.212.209:8080
NetLoc 18.53.35.179:80
NetLoc 114.187.128.212:80
NetLoc 60.103.18.131:80
NetLoc 172.132.76.194:443
NetLoc 113.67.58.224:80
NetLoc 20.179.35.232:80
NetLoc 73.249.184.108:80
NetLoc 9.222.103.137:443
NetLoc 204.197.26.221:443
NetLoc 224.138.203.45:80
NetLoc 244.157.143.47:80
NetLoc 190.67.48.224:80
NetLoc 180.42.36.109:80
NetLoc 208.118.116.55:80
NetLoc 4.195.63.225:25900
NetLoc 32.107.214.76:80
NetLoc 203.233.71.250:443
NetLoc 6.61.150.230:80
NetLoc 75.16.138.183:80
NetLoc 90.45.25.145:443
NetLoc 63.149.238.126:80
NetLoc 249.158.225.208:80
NetLoc 156.211.224.150:43912
NetLoc 229.210.208.203:80
NetLoc 27.219.195.210:80
NetLoc 30.255.153.175:80
NetLoc 216.69.26.86:80
NetLoc 182.180.65.173:443
NetLoc 197.45.165.116:443
NetLoc 79.101.37.210:80
NetLoc 12.25.99.130:80
NetLoc 50.56.242.72:8080
NetLoc 187.108.195.8:8080
NetLoc 212.219.93.114:443
NetLoc 138.4.86.20:8080
NetLoc 132.247.145.147:443
NetLoc 209.159.149.156:443
NetLoc 202.191.121.100:443
NetLoc 20.243.155.227:443
NetLoc 53.128.177.21:8080
NetLoc 235.250.233.187:80
NetLoc 35.214.161.230:443
NetLoc 34.5.168.186:443
NetLoc 210.147.248.235:443
NetLoc 254.220.78.226:47857
NetLoc 130.99.108.151:443
NetLoc 87.145.98.19:80
NetLoc 133.232.247.107:80
NetLoc 25.111.58.211:80
NetLoc 13.102.27.247:80
NetLoc 205.246.43.28:80
NetLoc 229.157.60.81:8080
NetLoc 180.168.197.23:80
NetLoc 29.156.163.20:443
NetLoc 53.44.118.111:80
NetLoc 123.100.180.115:43893
NetLoc 129.105.221.156:443
NetLoc 194.58.126.20:80
NetLoc 50.188.52.73:80
NetLoc 80.228.26.99:80
NetLoc 143.97.189.141:32240
NetLoc 241.174.170.164:28721
NetLoc 20.129.203.86:80
NetLoc 6.211.88.116:80
NetLoc 20.168.78.137:80
NetLoc 163.91.30.241:27879
NetLoc 174.120.121.230:39788
NetLoc 39.144.13.86:80
NetLoc 142.34.249.209:443
NetLoc 204.42.154.209:80
NetLoc 66.32.198.58:80
NetLoc 105.149.112.90:80
NetLoc 238.9.247.103:80
NetLoc 141.127.109.227:35000
NetLoc 250.5.29.204:80
NetLoc 232.245.197.186:80
NetLoc 8.218.248.66:80
NetLoc 97.215.155.187:80
NetLoc 138.196.78.240:80
NetLoc 173.126.49.27:443
NetLoc 84.22.102.112:80
NetLoc 145.89.215.87:8080
NetLoc 10.94.237.3:80
NetLoc 25.100.119.180:443
NetLoc 206.63.226.28:80
NetLoc 149.201.173.198:80
NetLoc 15.26.248.116:8080
NetLoc 218.5.226.178:80
NetLoc 245.187.185.226:80
NetLoc 90.251.34.209:443
NetLoc 65.159.238.36:443
NetLoc 30.184.131.202:443
NetLoc 103.216.152.95:80
NetLoc 34.58.82.4:80
NetLoc 249.167.103.219:47074
NetLoc 192.214.135.145:80
NetLoc 199.48.116.234:80
NetLoc 163.109.92.34:42753
